Source/WebCore/Modules/applepay/ApplePaySessionPaymentRequest.h

```diff
@@ -259,6 +269,9 @@ class ApplePaySessionPaymentRequest {
 #endif
 #if ENABLE(APPLE_PAY_LATER_AVAILABILITY)
             , m_applePayLaterAvailability(WTFMove(applePayLaterAvailability))
+#endif
+#if ENABLE(APPLE_PAY_MERCHANT_CATEGORY_CODE)
+            , m_merchantCategoryCode(WTFMove(merchantCategoryCode))
```

Comment:
Hmm, there's no point in moving an lvalue. Consider making `merchantCategoryCode` an rvalue reference to match the rest of the arguments.